pom.xml

```diff
@@ -88,8 +88,13 @@
 		<dependency>
 			<groupId>org.springframework.integration</groupId>
 			<artifactId>spring-integration-xml</artifactId>
+			<exclusions>
+				<exclusion>
+					<groupId>commons-logging</groupId>
+					<artifactId>commons-logging</artifactId>
+				</exclusion>
```

Comment:
Not sure in this exclusion because of:
```
./gradlew :spring-integration-xml:dependencies
...
compile - Dependencies for source set 'main'.
+--- project :spring-integration-core                                                                           
|    +--- org.springframework:spring-core:4.3.0.BUILD-SNAPSHOT
|    |    \--- commons-logging:commons-logging:1.2
|    +--- org.springframework:spring-aop:4.3.0.BUILD-SNAPSHOT
...
```
So, we have `commons-logging` as a transitive dependency from the `spring-core`. According to your logic we have to do the same exclusion from everywhere!
OTOH I don't see that anyone uses `commons-logging-api`.
But... I see that Spring Boot excludes `commons-logging` from many its dependencies, including `spring-boot-starter`: https://github.com/spring-projects/spring-boot/blob/master/spring-boot-dependencies/pom.xml#L257

So, the question is here anyway: what led you to exclude it only from SI-xml dep and nowhere else ?

Thank you!